### ISSUE TITLE
case sensitive AENS names

### ISSUE THREAD

**Issue by marc0olo** [aeternity/ae_mdw#114]
Currently it seems like the mdw handles AENS names case sensitive. This needs to be fixed.

**Query Microsoft.chain**
`https://mainnet.aeternity.io/mdw/name/Microsoft.chain`

**Response Microsoft.chain**
```json
{
	"active": true,
	"info": {
		"active_from": 164641,
		"auction_timeout": null,
		"claims": [
			4896956,
			4886286,
			4867032,
			4843196,
			4842596,
			4839488,
			4839186,
			4817148,
			4797824,
			4763211
		],
		"expire_height": 383002,
		"ownership": {
			"current": "ak_5z1fmzTKR1GA1P7qiLDCC1s3V7AK2RRpNbXqUhfHQbUeg7mmV",
			"original": "ak_5z1fmzTKR1GA1P7qiLDCC1s3V7AK2RRpNbXqUhfHQbUeg7mmV"
		},
		"pointers": {
			"account_pubkey": "ak_2QGAAqDXK7g8zCbck7zm25TGAW1hRuVCET2SRCCFCMSMjrVCrF"
		},
		"revoke": null,
		"transfers": [],
		"updates": [
			17065589,
			13654248,
			9992991,
			6685927
		]
	},
	"name": "Microsoft.chain",
	"previous": [],
	"status": "name"
}
```

**Query microsoft.chain**
`https://mainnet.aeternity.io/mdw/name/microsoft.chain`

**Response microsoft.chain**
```json
{
	"active": false,
	"info": {
		"active_from": 164188,
		"auction_timeout": null,
		"claims": [
			4863365
		],
		"expire_height": 214188,
		"ownership": {
			"current": "ak_5z1fmzTKR1GA1P7qiLDCC1s3V7AK2RRpNbXqUhfHQbUeg7mmV",
			"original": "ak_5z1fmzTKR1GA1P7qiLDCC1s3V7AK2RRpNbXqUhfHQbUeg7mmV"
		},
		"pointers": {},
		"revoke": null,
		"transfers": [],
		"updates": []
	},
	"name": "microsoft.chain",
	"previous": [],
	"status": "name"
}
```

**Comment by marc0olo** [comment]
is this closed via #149?

**Comment by skkw** [comment]
Fixed in https://github.com/aeternity/ae_mdw/releases/tag/1.0.4
